sci-libs/ceres-solver/ceres-solver-2.2.0.ebuild

```diff
@@ -28,6 +28,7 @@ BDEPEND="${PYTHON_DEPS}
 "
 RDEPEND="
 	dev-cpp/glog[gflags?,${MULTILIB_USEDEP}]
+	cuda? ( dev-util/nvidia-cuda-toolkit )
```

Comment:
Should this be `dev-util/nvidia-cuda-toolkit:=` instead?

Reply:
Yep. Fixing.